### PR TITLE
Add integration instructions for Swift Package Manager

### PR DESCRIPTION
**Pull request by rick-bt**
https://hack.in.backtrace.io/T15182
We expect to release this feature May 2, 2023

Question: what's up with this syntax?
:::note Make sure to specify use_frameworks! in your Podfile. ::: That doesn't seem to resolve?

### Description
Added support for SPM - swift Package Manager - to backtrace-cocoa. The customer has new way to integrate backtrace-cocoa, and this change details them.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist
- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by rick-bt** [comment]
Hey - I like the tabs for obj-c and swift. Such tabs could be used here for Xcode, Package.swift and CocoaPods. Can that be done?

File: docs/error-reporting/platform-integrations/ios/setup.md [hunk 1]
```diff
@@ -44,9 +44,23 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 - Xcode 10 or above
 
 ## Install the SDK
+You can use this SDK through either Swift Package Manager or CocoaPods. The SPM package can be integrated directly within Xcode or by editing your package's Package.swift file.<br>
+Choose one of the following integration methods.
 
-Use [CocoaPods](https://cocoapods.org/) to install the latest version of the reporting library. To use CocoaPods, add the [Backtrace pod](https://cocoapods.org/pods/Backtrace) to your `Podfile`:
+### Via Xcode
```
Comment (lysannep) [inline docs/error-reporting/platform-integrations/ios/setup.md:50]: @rick-bt ~ Is Xcode the default installation method?

**Comment by lysannep** [comment]
@rick-bt ~ Re: the syntax used for the [notes](https://docs.saucelabs.com/contributing/style-guide/#admonitions) uses Docusaurus Markdown, so something like: 
```
:::note
Relevant information for you.
:::
```

will render as: 
Screenshot 2023-05-01 at 2 23 06 PM

Docusaurus markdown is also used to generate the [tabs](https://docs.saucelabs.com/contributing/style-guide/#tabs).

File: docs/error-reporting/platform-integrations/ios/setup.md (hunk 1 above)
Comment (rick-bt) [reply docs/error-reporting/platform-integrations/ios/setup.md]: No, there is not a default installation method.

**Comment by rick-bt** [comment]
The mighty Docusaurus goes Roar!

**Review by lysannep — APPROVED** [review]

File: docs/error-reporting/platform-integrations/ios/setup.md (hunk 1 above)
Comment (lysannep) [reply docs/error-reporting/platform-integrations/ios/setup.md]: Thanks!

**Review by lysannep — APPROVED** [review]

**Pull request merged**